### PR TITLE
remove the change event object being passed to the callback

### PR DESCRIPTION
**Pull request by phated**
@contra this will require a major bump so we don't break anyone

### CODE REVIEW

File: test/main.js [hunk 1]
```diff
@@ -45,18 +45,14 @@ describe('glob-watcher', function() {
     });
   });
 
-  it('should return a valid file struct via callback', function(done) {
+  it('should not pass a file struct via callback', function(done) {
```
Comment (contra) [inline test/main.js]: should not pass any args via callback

File: test/main.js [hunk 2]
```diff
@@ -45,18 +45,14 @@ describe('glob-watcher', function() {
...
     mkdirp.sync(path.dirname(expectedName));
     fs.writeFileSync(expectedName, 'testing');
 
     var watcher = watch(fname, function(evt) {
-      should.exist(evt);
-      should.exist(evt.path);
-      should.exist(evt.type);
-      evt.type.should.equal('changed');
-      evt.path.should.equal(expectedName);
+      should.not.exist(evt);
```
Comment (contra) [inline test/main.js]: should verify that arguments.length is 0

**Comment by phated** [comment]
updated

**Comment by phated** [comment]
Weird that coveralls is reporting 100%, istanbul reported less locally.  Anyway, this has been updated

**Comment by contra** [comment]
Probably needs some doc updates right?

**Comment by phated** [comment]
@contra yep, just updated them now.

**Pull request merged**